database.html

```diff
@@ -75,10 +73,78 @@ <h1>MyMood Database Table</h1>
       </ul>
     </div>
     
+    <!-- Zebra Striping -->
+    <script>
+    function zebraStripe() {
+      // Go through all rows that are visible, filter old rows by odd and even
+      var $allRows = $('tr:visible');
+      var $oddRows = $allRows.filter(':odd');
+      var $evenRows = $allRows.filter(':even');
+      
+      // Remove old classes, then add new ones
+      $oddRows.removeClass('even').addClass('odd');
+      $evenRows.removeClass('odd').addClass('even');
+    };
+    </script>
+    
     <!-- Search Function -->
     <script>
-      search.init(["#database_table"]);
-      search.searchRows();
+    function search() {
+      // Declare variables 

```

Comment:
"// Declare variables" is an obvious comment making it redundant.
For variable names, whats tr and td? I'm guessing tr means tableRow but I'm not sure about td.